include/grpcpp/security/credentials.h

```diff
@@ -122,22 +109,29 @@ class ChannelCredentials : private grpc::internal::GrpcLibrary {
 /// \see https://grpc.io/docs/guides/auth.html
 class CallCredentials : private grpc::internal::GrpcLibrary {
  public:
+  ~CallCredentials() override;
+
   /// Apply this instance's credentials to \a call.
-  virtual bool ApplyToCall(grpc_call* call) = 0;
-  virtual grpc::string DebugString() {
-    return "CallCredentials did not provide a debug string";
-  }
+  virtual bool ApplyToCall(grpc_call* call);
```

Comment:
Do either  of these methods actually get overridden anywhere?  If not, maybe they don't need to be virtual.

This may technically be an API change, but in practice I'm not sure there's any way that anyone could have actually subclassed this in a useful way without using internal APIs.

Reply:
Please do the same for `DebugString()`.

Reply:
Done